Docs/Migration-INI.md

```diff
@@ -0,0 +1,854 @@
+# Migrating from older versions - INI configuration
+This guide uses [YR mod base](https://github.com/Starkku/cncnet-client-mod-base) configuration as an example. The majority of changes also applies to to non-YR client configurations.
+
+It is **highly recommended** to make a complete backup of your mod before starting.
+
+## Edit `ClientDefinitions.ini`
+1. Add `[Settings]->UnixLauncherExe=Launcher.sh` (script file name can be anything)
+2. Create `Launcher.sh` in root directory:
+```sh
+#!/bin/sh
+
+cd "$(dirname "$0")"
+dotnet Resources/Binaries/UniversalGL/clientogl.dll "$@"
+```
+3. Add these entries in `[Settings]` (fill with your required/forbidden mod files):
+```ini
+; Comma-separated list of files required to run the game / mod that are not included in the installation.
+RequiredFiles=
+; Comma-separated list of files that cannot be present to run the game / mod without problems.
+ForbiddenFiles=
+```
+
+## Edit `SkirmishLobby.ini`

```

Comment:
Could you add a small recap on what changes we are doing and why? Here especially but generally too.